src/components/LegacyHeading/__stories__/LegacyHeading.stories.mdx

```diff
@@ -22,7 +22,7 @@ export const metaSettings = createStoryMetaSettingsDecorator({
 });
 
 <Meta
```

Comment:
Passing `parameters: { status: 'deprecated' }` to the `Meta` component won't work in the Component level